roles/manage_vpc_peering/README.md

```diff
@@ -0,0 +1,105 @@
+# manage_vpc_peering
+
```

Comment:
Please update also https://github.com/redhat-cop/cloud.aws_ops#roles because it is not automatically generated.